### PR TITLE
CY-2894 Added ability to add widgets to predefined tabs

### PR DESCRIPTION
**Pull request by kubama**
Updated edit mode look:
image

### CODE REVIEW

File: app/components/AddWidgetModal.js [hunk 1]
```diff
@@ -215,7 +216,14 @@ export default class AddWidgetModal extends Component {
...
         const { canInstallWidgets, onWidgetInstalled } = this.props;
         const addWidgetBtn = (
-            <Button icon="bar chart" labelPosition="left" basic content="Add Widget" className="addWidgetBtn" />
+            <EditModeButton
+                icon="add"
+                labelPosition="left"
+                basic
+                content="Add Widget"
+                className="addWidgetBtn"
+                style={{ marginBottom: 15 }}
```
Comment (qooban) [inline app/components/AddWidgetModal.js:225]: As we already adding `style` here, I would move everything from `.addWidgetBtn` class in `styles.scss` here as well.
Class itself could still be useful (in Cypress tests), so we can keep it.

File: app/components/EditModeButton.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from './basic';
+import './EditModeButton.css';
+
+export default function EditModeButton({ className, ...rest }) {
+    return <Button className={`editModeButton ${className}`} {...rest} />;
```
Comment (qooban) [inline app/components/EditModeButton.js:7]: If I didn't miss something, we can move more `Button` props here. Looks like `icon="add" basic labelPosition="left"` are in common.

Comment (kubama) [reply app/components/EditModeButton.js:7]: Yes, I planned to do it later as this component is going to be reused in tabs management as well, so the set of common props may be eventually different.

File: app/components/AddWidgetModal.js (hunk 1 above)
Comment (kubama) [reply app/components/AddWidgetModal.js:225]: It turned out the styling was not applicable, so I just removed it.

**Pull request merged**